lib/json5/yojson_json5.mli

```diff
@@ -0,0 +1,101 @@
+module Safe : sig
+  type t = Yojson.Safe.t
+  type result := (t, string) Result.t
```

Comment:
I would try to avoid this alias, in case people unwisely do `open` since it clashes with the built-in `('a, 'e) result` type.